src/components/SidePanel/StartStop.tsx

```diff
@@ -73,7 +64,7 @@ export default () => {
                                     })
                                 ),
                         ]}
-                        onChangeComplete={completeChange}
+                        onChangeComplete={() => {}}
```

Comment:
This is optional, maybe not pass anything at all?

Reply:
It is not optional in the version of shared that I am using at the moment, but will try to remember to remove it after bumping shared :)